lib/index.js

```diff
@@ -154,8 +154,9 @@ const PSEUDO_HEADERS = {
  */
 function extractPseudoHeaders(params, requestOptions) {
   for(const key in Object.values(PSEUDO_HEADERS)) {
-    if(params[key]) {
-      requestOptions[key] = params[key];
+    const value = params[key];
+    if(value) {
```

Comment:
Is there any possibility that any of these headers might contain a javascript falsey value like 0, or false, or '' that might produce unexpected behavior here?

Reply:
GOOD CATCH! Addressed in https://github.com/digitalbazaar/http-signature-header/pull/13/commits/6628a7bf943945ecc077c059592e93ce168e06e4